### ISSUE TITLE
Create flow diagrams for all 

### ISSUE THREAD

**Issue by simkimsia** [simkimsia/label-ecommerce#14]
1. Anonymous Customer can checkout items
2. At checkout, Customer can remove or update the quantities of the items.
3. At checkout, Customer can see the internet banking details, so that Customer can pay via internet banking or cash on delivery.
4. At checkout, Customer can checkout via Paypal
5. At checkout, Customer can  fill in their names plus choose the font before checkout.
6. Each checkout will generate an Order with OrderLineItems in them.
7. Customer can choose to buy different amounts of same variants.
8. Customer contact will be stored in database for access by admin.
9. Shop Admin can update product images.
10. Shop Admin can create, update, delete products and variants.
11. Anonymous Customer can send enquiry emails to Shop Admin
12. Shop Admin can logout and login.
13. Shop Admin can reset own password.
14. There is a forget password and reset password functionality for Shop Admin.
15. Customers can navigate to view FAQ, Shop Front, About, Contact and the various Product Pages.

**Comment by simkimsia** [comment]
1. Anonymous Customer can checkout items  --> add product to cart
2. At checkout, Customer can remove or update the quantities of the items.
3. At checkout, Customer can see the internet banking details, so that Customer can pay via internet banking or cash on delivery. --> pay by iBanking
4. At checkout, Customer can checkout via Paypal --> pay by PayPal
5. At checkout, Customer can fill in their names plus choose the font before checkout. --> Fill in name and font at checkout
6. Each checkout will generate an Order with OrderLineItems in them. --> ShopAdmin can view Order with OrderLineItems
7. Customer can choose to buy different amounts of same variants. --> buy different amounts of variants of same product.
8. Customer contact will be stored in database for access by admin. --> Shop Admin view Customer details
9. Shop Admin can update product images.
10. Shop Admin can reorder the product images.
11. Shop Admin can create, update, delete products and variants.
12. Shop Admin can reorder the variants.
13. Anonymous Customer can send enquiry emails to Shop Admin
14. Shop Admin can logout and login.
15. Shop Admin can reset own password.
16. There is a forget password and reset password functionality for Shop Admin.
17. Customers can navigate to view FAQ, Shop Front, About, Contact and the various Product Pages. --> view static pages

**Comment by simkimsia** [comment]
DO use case first. Finish use case by Sunday.

**Comment by aikchun** [comment]
Point 14. Forget password includes the function to reset the password. Do you mean forget password and 'change'' password functionality for Shop Admin?

**Comment by simkimsia** [comment]
@AikChun For 14, split it into 2 separate ellipses.

Forget & reset password ( which is obviously for people who are unable to login)

Change password (which is for people already logged in)

Is that clearer?

**Comment by simkimsia** [comment]
Oh I see that my point 13 conflicts with 14. Okay. Let me rephrase again.

point 14 is 1 ellipse called
Forget & reset password ( which is obviously for people who are unable to login)

point 13 is 1 ellipse called 
Change password (which is for people already logged in)
